server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -111,7 +114,13 @@ private String getParentBuild(SRunningBuild build) {
         return String.valueOf(parentBuildPromotion.getId());
     }
 
-    private void setSpanBuildAttributes(SRunningBuild build, String buildName, Span span) {
+    private void setSpanBuildAttributes(SRunningBuild build, Span span) {
+        setCommonSpanBuildAttributes(build, span);
+        this.otelHelper.addAttributeToSpan(span, PluginConstants.ATTRIBUTE_SERVICE_NAME,  build.getBuildTypeExternalId());
```

Comment:
Why wouldn't we want these on all spans?

Reply:
Yep this PR does put them on all spans now! The only difference the spans have now are the `service_name` and `name` (spanName)

Reply:
been amalgamated into one method! Thanks for the hints!